DXRModules/DeusEx/Classes/DXRFlagsNGPMaxRando.uc

```diff
@@ -92,13 +93,17 @@ simulated function RandomizeSettings(bool forceMenuOptions)
     settings.enemiesshuffled = 100;
     MaxRandoVal(settings.enemies_nonhumans);
 
-    if(DXRFlags(self).IsHalloweenMode()) { // this cast is pretty nasty
-        settings.enemyrespawn = rng(10) + 15;
-    }
-    else if(chance_single(33)) {
-        settings.enemyrespawn = rng(120) + 120;
+    if(DXRFlags(self).IsHalloweenMode()) {
+        moresettings.reanimation = rng(10) + 15;
+    } else if (chance_single(33)) { // this cast is pretty nasty
```

Comment:
might as well do this slightly differently? maybe a little cleaner?
```suggestion
    } else if (chance_single(16)) {
        moresettings.reanimation = rng(20) + 30;
    } else if (change_single(16)) {
        settings.enemyrespawn = rng(120) + 120;
    }
```

Reply:
I think the way it is now is better because it makes it simpler to change the total odds. When I rebalance things in a separate PR, I'd like to change the total odds to below 33% and that would be a lot more straightforward the way I wrote it.

Reply:
Also, the odds aren't the same with the numbers you used. Reanmiation has a 16% chance, while respawn has a 13.44% chance.